src/test/ensemble_tests.cpp

```diff
@@ -88,11 +89,11 @@ class EnsembleFlowTest : public ::testing::Test {
     }
 
     void performWrongPipelineConfigTest(const char* configFileContent) {
-        std::string fileToReload = "/tmp/ovms_config_file1.json";
```

Comment:
Future improvement: move config.json creation in separate fixture aswell, there are plenty of unit tests that create config.json and startFromFile. This way you would not need to remember to create config.json in `std::string fileToReload = directoryPath + "/ovms_config_file1.json";` place.

Reply:
Yes - exactly the same issue while working on future MR. I think the cleanest way would be to remove this method from global scope and put it in TestWithTempDir. Then path given to this method would by default use directoryPath.